api/v1alpha1/feature_flags.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 Dynatrace LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"strconv"
+)
+
+const (
+	AnnotationFeaturePrefix                   = "alpha.operator.dynatrace.com/feature-"
+	AnnotationFeatureDisableActiveGateUpdates = AnnotationFeaturePrefix + "disable-activegate-updates"
+	AnnotationFeatureDisableHostsRequests     = AnnotationFeaturePrefix + "disable-hosts-requests"
+	AnnotationFeatureOneAgentMaxUnavailable   = AnnotationFeaturePrefix + "oneagent-max-unavailable"
```

Comment:
Exported on purpose? Is the idea to use them in tests?

Reply:
Nope. Thanks for letting me know.